src/map.rs

```diff
@@ -794,6 +794,46 @@ where
         });
     }
 
+    /// Search over a sorted map with a comparator function.
+    ///
+    /// Returns the position where that value is present, or the position where can be inserted to maintain the sort.
+    /// see [slice::binary_search_by] for more details.
+    /// **O(log(n))**
```

Comment:
Polishing like this is applicable to most of the docs.
```suggestion
    /// Returns the position where that value is present, or the position where it can be inserted to maintain the sort.
    /// See [`slice::binary_search_by`] for more details.
    ///
    /// Computes in **O(log(n))** time.
```